<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDatatypeNameWidget</class>
 <widget class="QWidget" name="EditDatatypeNameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>166</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="boolRadioButton">
       <property name="text">
        <string>bool</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="integerRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>integer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string> range </string>
       </property>
       <property name="buddy">
        <cstring>rangeFromLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rangeFromLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string> to </string>
       </property>
       <property name="buddy">
        <cstring>rangeToLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rangeToLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="otherRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>other</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="otherComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="setCheckBox">
     <property name="text">
      <string>set of</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>setCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotSomethingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boolRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotSomethingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>integerRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotSomethingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotSomethingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rangeFromLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotSomethingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rangeToLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotSomethingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherComboBox</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotSomethingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>integerRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotIntegerRadioButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditDatatypeNameWidget</receiver>
   <slot>slotOtherRadioButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSomethingChanged()</slot>
  <slot>slotIntegerRadioButtonToggled(bool)</slot>
  <slot>slotOtherRadioButtonToggled(bool)</slot>
 </slots>
</ui>
